### DISCUSSION TITLE
Incorrect warm white color for Virtual CT (RGBW bulb)

### DISCUSSION THREAD

**Discussion by ferbulous** [arendst/Tasmota#19182] [Q&A]
Hi, I have an RGBW (cold white) bulb, I've enabled:

```setoption105 1```
```setoption106 1```

As for the virtual CT
I use ```setoption 107 0``` to emulate warm white,
However, this seems to reverse the warm & cold white on the color picker (Cold white output when selecting warm white and vice versa)
so I changed to ```setoption 107 1``` to rectify that.

The emulated warm white seems to be blue tint instead of yellow color. How can I modify this?
If I need to use RGBWWTable, is there an example for that?

**Comment by s-hadinger** [comment]
Yes, but it's been a while since I introduced this, so I don't remember all the details.

Except that emulating warm white doesn't work very well and often leads to a severe loss in brightness.

**Reply by ferbulous** [reply]
Hi, is there example for using RGBWWTable to change the blue tint to yellow-ish?
For the brightness loss does that happen over time?

**Reply by s-hadinger** [reply]
The best approach is first to calibrate the lights. Return to standard configuration, and try different `Color FFFFFF00`(white with RGB) and `Color 000000FF` (color with W). Then try to find the right combination of RGB and W to meet the virtual white you want.

My comment on brightness is that very often `Color FFFFFF00` is less bright than `Color 000000FF` which means that you can't have a warm white as bright as cold white - due to the nature of leds.
